repos/rust-lang/pontoon.toml

```diff
@@ -0,0 +1,8 @@
+org = "rust-lang"
+name = "pontoon"
+description = "Localization platform used by Rust"
+bots = []
+
+[access.teams]
+community = "triage"
```

Comment:
I'm not sure if this is necessary as the community team isn't really active (outside of TWiR).